pkg/cloud/services/compute/instance_types.go

```diff
@@ -102,10 +102,7 @@ func (is *InstanceStatus) UpdateBastionStatus(openStackCluster *infrav1.OpenStac
 		openStackCluster.Status.Bastion = &infrav1.BastionStatus{}
 	}
 
-	openStackCluster.Status.Bastion.ID = is.ID()
 	openStackCluster.Status.Bastion.Name = is.Name()
-	openStackCluster.Status.Bastion.SSHKeyName = is.SSHKeyName()
```

Comment:
with SSHKeyName being removed, the function at line 91 can also be removed.

Reply:
Also, it's worth documenting that these fields are being removed

Reply:
Both addressed. Also noticed that we weren't using `AvailabilityZone()` any more and removed that, too.